source/Plugins/SymbolFile/DWARF/DWARFASTParserSwift.cpp

```diff
@@ -120,14 +120,6 @@ lldb::TypeSP DWARFASTParserSwift::ParseTypeFromDWARF(const SymbolContext &sc,
     }
     if (SwiftLanguageRuntime::IsSwiftMangledName(name.GetCString()))
       mangled_name = name;
-    else {
-      const char *type_name_cstr = name.GetCString();
```

Comment:
I have actually seen instances on types without mangled names in recently generated DWARF, however they were always forward declarations that only existed to serve as the DeclContext of a function. So I still think this is safe.